build/dotenv/test.env

```diff
@@ -0,0 +1,15 @@
+TEST_DATABASE_MYSQL_DSN=mmuser:mostest@tcp(mysql:3306)/mattermost_test?charset=utf8mb4,utf8\u0026readTimeout=30s\u0026writeTimeout=30s
```

Comment:
While we are here, let's also unescape `\u0026` to just `&` to make things a little bit cleaner.